app/index.jsx

```diff
@@ -124,6 +138,17 @@ class App extends Component {
             || JSON.stringify(this.state.settings) !== JSON.stringify(nextState.settings);
     }
     */
+    cleanComparison = () => {
+        try {
+            const layout_string = localStorage.getItem(`layout-${chartId}`),
+                layout = JSON.parse(layout_string !== '' ? layout_string : '{}');
+
+            layout.symbols.splice(1, layout.symbols.length - 1);
+            localStorage.setItem(`layout-${chartId}`, JSON.stringify(layout));
+        } catch (e) {
```

Comment:
why do you need this silent try catch block here?

also I think `removeAllComparisons` is a better name for this method.